diff --git a/build/index.d.ts b/build/index.d.ts
index 886800935ed0d85f0e6396730f8e2a335467d7fa..da1871a106fd3d9655fe26f359257b9ba9773376 100644
--- a/build/index.d.ts
+++ b/build/index.d.ts
@@ -11,6 +11,10 @@ export declare function run(
   project?: string,
 ): Promise<void>;
 
+export declare function buildArgv(
+  maybeArgv?: Array<string>,
+): Promise<Config.Argv>;
+
 export declare const yargsOptions: {
   [key: string]: Options;
 };
diff --git a/build/index.js b/build/index.js
index fc00c8bfb4843380c6168e2bd54040a54e2470a7..92e51c4551be436903b281df95f1610a69e87cf1 100644
--- a/build/index.js
+++ b/build/index.js
@@ -15,5 +15,11 @@ Object.defineProperty(exports, 'yargsOptions', {
     return _args.options;
   }
 });
+Object.defineProperty(exports, 'buildArgv', {
+  enumerable: true,
+  get: function () {
+    return _run.buildArgv;
+  }
+});
 var _run = require('./run');
 var _args = require('./args');
