diff --git a/build/index.js b/build/index.js
index 0ba01789d77b081f2cbdaa689e851ad94e221820..720d5209265e12e73c6d0866abbd92751aad4ddb 100644
--- a/build/index.js
+++ b/build/index.js
@@ -2006,6 +2006,7 @@ class Runtime {
       return jestObject;
     };
     const jestObject = {
+      getIsolatedModuleRegistry: () => this._isolatedModuleRegistry,
       advanceTimersByTime: msToRun =>
         _getFakeTimers().advanceTimersByTime(msToRun),
       advanceTimersByTimeAsync: async msToRun => {
