diff --git a/node_modules/@types/jest/index.d.ts b/node_modules/@types/jest/index.d.ts
index b730d29..ada071d 100755
--- a/node_modules/@types/jest/index.d.ts
+++ b/node_modules/@types/jest/index.d.ts
@@ -112,6 +112,7 @@ interface FakeTimersConfig {
 }
 
 declare namespace jest {
+    function getIsolatedModuleRegistry(): Map<string, any> | null;
     /**
      * Disables automatic mocking in the module loader.
      */
